app/scripts/modules/core/src/presentation/text/BreakString.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+
+import * as styles from './BreakString.module.css';
+
+export interface IBreakStringProps {
+  className?: string;
+  children?: React.ReactNode;
+}
+
+export const BreakString = ({ className, children }: IBreakStringProps) => (
+  <span className={classNames(styles.breakString, className)}>{children}</span>
```

Comment:
I know this is kind of blasphemous, but...what if you inlined the styles on the `<span>` here? That would prevent someone from overriding  the `word-break`, `overflow-wrap`, `hyphens` values - and avoid confusion over ambiguous stylesheet insertion order, i.e. if I have a class I send into `<BreakString>` that declares `hyphens: auto`, it might override this value, it might not, depending on which stylesheet goes into the DOM at which location.

Unless we know that these styles will be inserted in a predictable order, it might be worth it just to provide that guarantee. Or not! Maybe it's an issue not really worth worrying about.

Reply:
I thought about this for a couple of microseconds and then muscle memory kicked in and I did the usual thing — I totally agree and hadn't considered the potential of override wonkiness. Doing now.